code_samples/page/react_app_block/config/packages/react_blocks.yaml

```diff
@@ -0,0 +1,11 @@
+ibexa_fieldtype_page:
+    react_blocks:
+        calculator:
+            name: Calculator
+            category: Demo
+            thumbnail: /bundles/ibexaicons/img/all-icons.svg#date
+            component: Calculator
+            attributes:
+                a:
+                    type: integer
+                b: integer
```

Comment:
Shouldn't be there are also:
type: integer for b?
@julitafalcondusza @adriendupuis

Reply:
@julitafalcondusza 
It illustrates the following statement bellow: "Both name and attributes support a short syntax and a long one for specifics."

Reply:
@juskora @julitafalcondusza 
It looks a bit weird in this example because only `type` param is used.
I tried few things I thought would be available params but it doesn't care.
Let's keep it like this for this PR.